### ISSUE TITLE
format doesn't work

### ISSUE THREAD

**Issue by cawa-93** [SoapBox/linkifyjs#91]
```javascript
$('<span>http://github.com/SoapBox/linkifyjs/search/?q=this+is+a+really+long+query+string</span>').linkify({
	format: function (value, type) {
		console.log(value);
		return 'link';
	}
}).html();
```
Result:
```HTML
"<a href="http://github.com/SoapBox/linkifyjs/search/?q=this+is+a+really+long+query+string" class="linkified" target="_blank">http://github.com/SoapBox/linkifyjs/search/?q=this+is+a+really+long+query+string</a>"
```

The message in console also does not appear

**Comment by nfrasser** [comment]
Which version of Linkify are you using? I tried this in Chrome and got this

```
"<a href="http://github.com/SoapBox/linkifyjs/search/?q=this+is+a+really+long+query+string" class="linkified" target="_blank">link</a>"
```

As well as the console message.

**Comment by cawa-93** [comment]
Installed using bower
jQuery-linkify#1.1.7 (latest is 2.0.0-beta.5)
└── jquery#2.1.4
![2015-12-03 20 33 41](https://cloud.githubusercontent.com/assets/1662812/11569933/290c801c-99fd-11e5-8697-16391c120048.png)

![2015-12-03 20 29 55](https://cloud.githubusercontent.com/assets/1662812/11569827/a8bee12a-99fc-11e5-89d9-13e08f01762b.png)
![2015-12-03 20 29 39](https://cloud.githubusercontent.com/assets/1662812/11569826/a8ba4f98-99fc-11e5-84b2-51a8d0d9fd7f.png)

**Comment by nfrasser** [comment]
The `format` option is only available in 2.0.

Try this instead:

```
bower install linkifyjs#v2.0.0-beta.5
```
